tests/functional/pages/surveys/relationship/household-composition.page.js

```diff
@@ -0,0 +1,62 @@
+// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
```

Comment:
It doesn't look like this is imported anywhere, so I think it can be safely removed.